### DISCUSSION TITLE
界面记录这个是否可以增加一个开关？

### DISCUSSION THREAD

**Discussion by fansangg** [gkd-kit/gkd#692] [Q&A] [answered]
佬，你觉得是否有必要增加一个开关，由用户决定需不需要开启这个功能和入口，个人感觉大部分用户应该不太需要这个功能
@lisonge

**Comment by lisonge** [comment, marked as answer]
https://github.com/gkd-kit/gkd/commit/ae0d7c7cf5b61c3512e1f213c3e1eb365c8e80ab
